ansible/roles/gitlab/tasks/main.yml

```diff
@@ -112,4 +112,4 @@
     dest: '/etc/ansible/facts.d/gitlab.fact'
     owner: 'root'
     group: 'root'
-    mode: '0644'
+    mode: '0755'
```

Comment:
The `gitlab.fact` file is not a script at the moment, so this is unnecessary and might even break Ansible which will try to execute it blindly, I suspect.